File: yunshin/4주차/BruteForce/Programmers/86491_최소직사각형.py
def solution(sizes):
    # 큰 값을 가로로, 작은 값을 세로로
    maxWidth = -1
    maxHeight = -1
    for width,height in sizes:
        if width >= height:
            maxWidth = max(maxWidth,width)
            maxHeight = max(maxHeight,height)
        else:
            maxWidth = max(maxWidth,height)
            maxHeight = max(maxHeight,width)
            
    return (maxHeight*maxWidth)

File: yunshin/4주차/BruteForce/Programmers/86491_최소직사각형.js
function solution(sizes) {
  let maxWid = 0
  let maxHei = 0
  sizes.forEach((size) => {
    let [width, height] = size
    if (width > height) {
      maxWid = Math.max(maxWid, width)
      maxHei = Math.max(maxHei, height)
    } else {
      maxWid = Math.max(maxWid, height)
      maxHei = Math.max(maxHei, width)
    }
  })
  return maxWid * maxHei
}
